build.ps1

```diff
@@ -1,3 +1,7 @@
+dotnet nuget locals global-packages --clear
```

Comment:
I realize this is pretty agressive for some reason changes to the json logger nuget package weren't being picked up when this script ran

Reply:
Gave that a try. My powershell is really weak though so its not elegant.